operations/mimir-mixin/config.libsonnet

```diff
@@ -78,6 +78,115 @@
       gateway: helmCompatibleName('(gateway|cortex-gw|cortex-gw).*'),
     },
 
+    deployment_type: 'container',
```

Comment:
I would call it `kubernetes` (instead of `kubernetes`) since some panels actually use K8S metrics. WDYT?

Reply:
indeed, I've made the changes accordingly.